incuna-sass/functions/_rem.sass

```diff
@@ -5,6 +5,8 @@ $use-rem: false !default
     @if $use-rem
         @if $target == 0
             @return 0
+        @if $target == 1px
+            @return 1px
```

Comment:
IIRC, `2px` was also a problem? Perhaps we could check `$target` for these values on line 5, returning `$target` without modification on line 12 if it's in the blacklist?